int32 id
sensor_msgs/CompressedImage img
int64[] obj_ids
BoundingBox[] obj_bboxes
string result